uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/instances/leave-request-instance.js

```diff
@@ -123,9 +123,12 @@ define([
             comments: [],
             files: [],
             request_type: 'leave',
-            uploader: FileUpload.uploader({
+            //FileUpload.uploader has uploader property which was causing circular reference issue
+            //hence renamed this uploader to fileUploader
+            fileUploader: FileUpload.uploader({
               entityTable: 'civicrm_hrleaveandabsences_leave_request',
-              crmAttachmentToken: sharedSettings.attachmentToken
+              crmAttachmentToken: sharedSettings.attachmentToken,
+              queueLimit: 5 //allow to upload upto 5 files for now
```

Comment:
this comment is unnecessary and kind of confusing with that "for now", it seems this is temporary?